src/mem.rs

```diff
@@ -0,0 +1,60 @@
+// NOTE Copied verbatim from the rlibc crate
```

Comment:
I don't feel too strongly about it either way. The point of making these symbols weak is that it allows you to override them with your own implementations if you want.